iocore/net/quic/QUICStream.cc

```diff
@@ -340,6 +333,10 @@ QUICStream::recv(const QUICStreamFrame &frame)
     return error;
   }
 
+  if (this->_received_stream_frame_buffer.has_all_data()) {
+    this->_state.update_on_transport_recv_event();
```

Comment:
Call transport_recv_event when receiving all data.

Reply:
>If the frame has fin flag, it will transit to SizeKnow immediately. So we only check whether the the state is SizeKnown.

Because we have all data we need to signal the state to transit to terminal state